flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/table/batch/compact/BatchPartitionCommitterSinkTest.java

```diff
@@ -127,7 +127,7 @@ public Long timestamp() {
             };
 
     private static RuntimeContext getMockRuntimeContext() {
-        return new MockStreamingRuntimeContext(false, 0, 0) {
+        return new MockStreamingRuntimeContext(false, 1, 0) {
```

Comment:
Unrelated change

Reply:
These tests are conflicted with the newly added correctness check in the `TaskInfoImpl`. For example, the subtask ID shouldn't be large than or equal to the operator parallelism number. I've fixed these errors in the commit `[hotfix] Fix the logical errors for the tests related to task metadata`.